bin/poll.js

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+
+const {
+  backends,
+  customResourceManifest,
+  kubeClient,
+  logger
+} = require('../config')
+
+async function main () {
+  const destinationPath = process.env['DESTINATION_PATH']
+  const externalSecretName = process.env['EXTERNAL_SECRET_NAME']
+  const namespace = process.env['EXTERNAL_SECRET_NAMESPACE']
```

Comment:
Would it be worth throwing an exception if any of these aren't provided, to help debug (by offering a specific error message)?